charts/barman/templates/statefulset.yaml

```
@@ -23,18 +23,34 @@ spec:
         - name: barman
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - "/bin/sh"
+                  - "-c"
+                  - "HOME=/home/barman; cp /.pgpass $HOME/; chown barman:barman $HOME/.pgpass; chmod 600 $HOME/.pgpass"
```

Consider using more `&&` instead of `;` or do a `set -e`...
```suggestion
                  - "HOME=/home/barman; cp /.pgpass $HOME/ && chown barman:barman $HOME/.pgpass && chmod 600 $HOME/.pgpass"
```